app/models/legal_aid_application.rb

```diff
@@ -0,0 +1,13 @@
+class LegalAidApplication < ApplicationRecord
+
+
+  before_create :create_app_ref
+
+
+
+  private
+
+   def create_app_ref
+     self.application_ref = SecureRandom.uuid
```

Comment:
A note that we might want to try [this approach](23891181-62ef-4e27-ad6d-c13d2159ee81) when we move to postgres.

Reply:
if this approach is a url link it doesn't go anywhere other than a 404